bounties/npm/node-resourcehacker/1/vulnerability.json

```diff
@@ -0,0 +1,50 @@
+{
+	"PackageVulnerabilityID": 1,
+	"DisclosureDate": "2020-08-17",
+	"AffectedVersionRange": "*",
+	"Summary": "Resources Downloaded over Insecure Protocol",
+	"Contributor": {
+		"Discloser": "",
+		"Fixer": ""
+	},
+	"Package": {
+		"Registry": "npm",
+		"Name": "resourcehacker",
+		"URL": "https://www.npmjs.com/package/resourcehacker",
+		"Downloads": "1936"
+	},
+	"CWEs": [
+		{
+			"ID": "CWE-494",
+			"Description": ""
+		}
+	],
+	"CVSS": {
+		"Version": "3.1",
+		"AV": "N",
+		"AC": "H",
+		"PR": "L",
+		"UI": "R",
+		"S": "U",
+		"C": "H",
+		"I": "H",
+		"A": "H",
+		"E": "",
+		"RL": "",
+		"RC": "",
+		"Score": "7.1"
+	},
+	"CVEs": [
+		"CVE-2016-10646"
+	],
+	"Repository": {
+		"URL": "https://github.com/felicienfrancois/node-resourcehacker",
+		"Codebase": [
+			"JavaScript"
+		]
+	},
+	"Permalinks": [
+		"https://github.com/felicienfrancois/node-resourcehacker/blob/13f71e9e0c46788f6eff5a7e82b6fa11a0a3dde4/scripts/install.js#L4"
```

Comment:
Just replace with the `master` permalink :smile: (https://github.com/felicienfrancois/node-resourcehacker/blob/master/scripts/install.js#L4)

Reply:
Permalink updated, cheers 😄